src/platform/imx8ulp/include/platform/trace/trace.h

```diff
@@ -0,0 +1,22 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2019 Intel Corporation. All rights reserved.
+ *
+ * Author: Tomasz Lauda <tomasz.lauda@linux.intel.com>
+ */
```

Comment:
same here. If you add a new file (even if it is copy pasted) please add your name and NXP copyright.
You can also mentioned the initial file you based your copy on .